code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm

```diff
@@ -137,10 +137,11 @@ Difficulty: Medium
 	INVOKE_ASYNC(src, .proc/lava_pools, amount)
 	swoop_attack(FALSE, target, 1000) // longer cooldown until it gets reset below
 	fire_cone()
-	sleep(10)
-	fire_cone()
-	sleep(10)
-	fire_cone()
+	if(health < maxHealth/2)
```

Comment:
`*0.5` rather than `/2`

Reply:
Why?

Reply:
multiplication is preferred over division for ???

Reply:
Should I edit the other occurences of this in the file then? I was just taking from what was there.